.gitmodules

```diff
@@ -58,7 +58,8 @@
 	url = https://github.com/ceph/spawn.git
 [submodule "src/pybind/mgr/rook/rook-client-python"]
 	path = src/pybind/mgr/rook/rook-client-python
-	url = https://github.com/ceph/rook-client-python.git
+	url = https://github.com/sebastian-philipp/rook-client-python.git
```

Comment:
@sebastian-philipp @josephsawaya does it make sense to update ceph/rook-client-python to be in sync with sebastian-philipp/rook-client-python ?

Reply:
There is one bit of information missing in the PR description: We're waiting for https://github.com/rook/rook-client-python/pull/4 to get merged. using my fork is just a stop-gap, till this PR is merged, which is blocked by https://github.com/rook/rook/pull/8122

Reply:
PR is merged. we can move over to rook/rook-client-python now

Reply:
I've moved it over to [rook/rook-client-python](https://github.com/rook/rook-client-python) in my latest commit